buildlib/fedora.Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN dnf install -y \
     file \
     gcc-c++ \
     git \
+    git-clang-format \
```

Comment:
are you test it? I think need add `clang` package

Reply:
The package is `git-clang-format`. `clang` is already there (above). Tested locally.